pwndbg/commands/telescope.py

```diff
@@ -183,24 +191,37 @@ def collapse_repeating_values() -> None:
             result.extend(collapse_buffer)
         collapse_buffer.clear()
 
+    bp = pwndbg.gdblib.regs[pwndbg.gdblib.regs.frame]
+
+    def regs_or_frame_offset(addr):
```

Comment:
Each execution of the command will re-create the function object; shall we maybe put it in global scope ( ? )